tests/ha/test_data_pod_failure.py

```diff
@@ -938,68 +903,54 @@ def test_writes_during_pod_down(self):
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Services of pod %s are in offline state", pod_name)
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown data pod %s. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
 
-        pod_list.remove(pod_name)
-        LOGGER.info("Step 5: Check services status on remaining pods %s", pod_list)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=pod_list, fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services status on remaining pod are in online state")
-        event.clear()
         thread.join()
         responses = dict()
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
-        LOGGER.info("Step 6: Verify status for In-flight WRITEs while pod going down is failed.")
+        LOGGER.info("Step 3: Verify status for In-flight WRITEs while pod going down is failed.")
         pass_logs = list(x[1] for x in responses["pass_res"])
         fail_logs = list(x[1] for x in responses["fail_res"])
         resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
-        assert_utils.assert_false(len(resp[1]), f"Expected Pass, But Logs which contain failures:"
-                                                f" {resp[1]}")
+        assert_utils.assert_false(len(resp[1]),
+                                  f"Expected Pass, But Logs which contain failures: {resp[1]}")
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
                                  f"Logs which contain passed IOs: {resp[1]}")
-        LOGGER.info("Step 6: Verified status for In-flight WRITEs while pod going down is failed.")
+        LOGGER.info("Step 3: Verified status for In-flight WRITEs while pod going down is failed.")
 
-        LOGGER.info("STEP 7: Perform IOs with variable object sizes on degraded cluster")
         if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Step 4: Perform WRITEs-READs-Verify-DELETEs with variable object sizes "
+                        "on degraded cluster with new user")
             users = self.mgnt_ops.create_account_users(nusers=1)
             self.test_prefix = 'test-26441-1'
             self.s3_clean.update(users)
-        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, skipcleanup=True)
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
```

Comment:
as per discussion , can u please change the code where we have specific READ/WRITE tests?